controller/metrics/metrics.go

```diff
@@ -363,6 +363,9 @@ func boolFloat64(b bool) float64 {
 }
 
 func (c *appCollector) normalizeAppTargetRevision(in string) string {
+	if len(in) == 0 {
+		in = "head"
```

Comment:
nitpicking, but git semantics use all caps `HEAD`, not lowercase. lowercase head is nothing special for git.

Reply:
I have hesitated, meanwhile it's good to have prom labels lowercased right?